hooks/templatedatavalidator.js

```diff
@@ -0,0 +1,114 @@
+var fs = require('fs');
+var path = require('path');
+const { parseJamboConfig } = require('../commands/helpers/utils/jamboconfigutils');
+const { error } = require('../commands/helpers/utils/logger');
+/**
+ * Validates page template's data configuration
+ * 
+ * @param {Object} pageData data that gets passed into a page template
+ * @returns {boolean} false if validator should throw an error
+ */
+module.exports = function (pageData) {
+  const jamboConfig = parseJamboConfig();
+  return isGlobalConfigValid(pageData.global_config) & isPageVerticalConfigValid(pageData, jamboConfig);
```

Comment:
bitwise and is cool here! I feel like it might get us in trouble if we have future validators that return truthy/falsy values instead of actual booleans. Could we find a different way to write this?

Reply:
updated!